packages/react/src/components/ComboBox/ComboBox.story.jsx

```diff
@@ -54,6 +54,12 @@ const props = () => ({
   invalidText: text('Invalid text (invalidText)', 'A valid value is required'),
   size: select('Field size (size)', sizes, undefined) || undefined,
   onChange: action('fired onChange'),
+  menuFitContent: boolean('Menu fits content (menuFitContent)', false),
+  horizontalDirection: select(
+    'Horizontal direction (horizontalDirection)',
+    ['left', 'right'],
```

Comment:
I would avoid "right" and "left" or else it'll be hard to implement RTL support.  ie is "right" still right in RTL or does right mean left in RTL which is confusing..  Normally we try to use "start" and "end" so that when we enter RTL things that were LEFT (start) now become RIGHT(start) in RTL.

Reply:
Good point, I'll fix